### ISSUE TITLE
Bug: initial value for global search doesn't work

### ISSUE THREAD

**Issue by nullday** [rubanraj54/vue-bootstrap4-table#54]
Hey there!

I try to assign initial value for search input:
```
config: {
                    ...
                   global_search: {
                        init: {
                            value : "Christin"
                        }                        
                    },
                    ...
                }
```
But the actual input value is empty, like there is no initial value.
`data.global_search.init.value` - is correct.
`data.query.global_search` - is fine as well.

I've tried to fix it locally inside your library and it worked. 
I've just added $nextTick, like this:
```
this.$nextTick(() => {
                   this.$refs.global_search.value = this.global_search.init.value;
});
```

**Comment by rubanraj54** [comment]
Hi @nullday 

This is a strange issue, it is working perfectly in my local machine and not working in the codepen example.

I guess it depends on the different machine's power, so I fixed it by calling initGlobalSearch() function inside $nextTick. Now I can see the init value in the global search input box in all different machines.

Thanks for reporting this bug and it is fixed in the new release (1.1.9)

Cheers,
Ruby.

**Comment by nullday** [comment]
That was a fast fix!
Thanks.
